keymaps/templates/Apple Mail.cson

```diff
@@ -4,16 +4,39 @@
 # http://dashkards.com/img/dashkard-mail.png
 
 'body':
+
+  ### Jumping ###
+  'cmd-1' : 'navigation:go-to-inbox'
+  'cmd-2' : 'navigation:go-to-starred'
+  'cmd-3' : 'navigation:go-to-sent'
+  'cmd-4' : 'navigation:go-to-drafts'
+  'cmd-5' : 'navigation:go-to-all'
+  'cmd-6' : 'navigation:go-to-contacts'
+  'cmd-7' : 'navigation:go-to-tasks'
+  'cmd-8' : 'navigation:go-to-label'
+
+    ### Threadlist selection ###
+  'cmd-a': 'multiselect-list:select-all'
+
+
+  ### Application ###
+
+  'cmdopt-f': 'application:focus-search'
```

Comment:
I think this needs to be `cmdctrl-alt-f` to work properly. It might also be down at the bottom?